django/core/management/commands/loaddata.py

```diff
@@ -223,6 +245,39 @@ def save_obj(self, obj):
             self.objs_with_deferred_fields.append(obj)
         return saved
 
+    def bulk_create_obj(self, objects):
+        """
+        split objects into groups with the same model, then use bulk_create to load
+        all instances in a single statement
+        """
+        objects_in_fixture = 0
+        loaded_objects_in_fixture = 0
+        group_model = None
+        instances = []
```

Comment:
This is missing all the `excluded_models` and `excluded_apps` logic from `save_obj`.

Same thing with `allow_migrate_model`.